src/definitions/modules/dropdown.js

```diff
@@ -3523,6 +3526,9 @@ $.fn.dropdown = function(parameters) {
             if(module.is.verticallyScrollableContext()) {
               calculations.menu.offset.top += calculations.context.scrollTop;
             }
+            if(module.has.subMenu()) {
```

Comment:
Submenu has alto to respect multi nested submenus
```suggestion
            if(module.has.subMenu($currentMenu)) {
```